orderbook/src/database/events.rs

```diff
@@ -138,7 +138,8 @@ async fn insert_invalidation(
     event: &Invalidation,
 ) -> Result<(), sqlx::Error> {
     const QUERY: &str =
-        "INSERT INTO invalidations (block_number, log_index, order_uid) VALUES ($1, $2, $3);";
+        "INSERT INTO invalidations (block_number, log_index, order_uid) VALUES ($1, $2, $3) \
```

Comment:
Can we add a comment on why we do the on conflict do nothing here and below (maybe just a link to the issue).

Reply:
added